geiger/translation.py

```diff
@@ -0,0 +1,26 @@
+# Imports the Google Cloud client library
+from googletrans import Translator
+
+def detect_lang(text):
+    # detects language of text
+    pass
+
+
+def translate_to_target(translator, text, src, dest):
+    return translator.translate(text,src=src,dest=dest)
+
+
+def translate_text(text):
+    """ Translates string from src to each language in dest
+    Translate back to english.
+    returns list of strings
+    """
+    src = 'en'
+    langs = ['nl','de','it','fr']
+    translator = Translator()
+    trans = [translate_to_target(translator, text, src, l) for l in langs]
```

Comment:
I think these nested loops are a bit hard to understand and also overriding the `trans` variable every time seems a bit dangerous. for me a better approach after setting the original `trans` variable is to do something like:

```
english_variations = set()
for tran in trans:
    if tran: # watch out for nones
         translate_back_to_english_here
          if english translation:
                  add english translations to set.

```